plugins/faustwp/includes/telemetry/functions.php

```diff
@@ -103,3 +103,18 @@ function get_plugin_version() {
 function get_wpgraphql_content_blocks_plugin_version() {
 	return defined( 'WPGRAPHQL_CONTENT_BLOCKS_VERSION' ) ? WPGRAPHQL_CONTENT_BLOCKS_VERSION : null;
 }
+
+/**
+ * Returns the anonymous client id for this site that has opted in for telemetry
+ */
+function get_telemetry_client_id(): string|null  {
+	/**
+	 * @TODO Upon saving the site's telemetry decision, if they accept, we'll
+	 * also need to generate a unique, anonymous client ID for them to be sent 
+	 * with GA requests.
+	 * 
+	 * If a string is returned, telemetry is enabled and a client id has been generated.
+	 * If this function returns null, either telemetry is off, or a client ID is not created.
+	 */
+ return null;
+}
```

Comment:
@TeresaGobble When we save a user's preferences for their telemetry opt-in, we'll also want to generate a unique id for them (something like a UUID) as GA requires this. You can take a look at our previous implementation in the CLI for inspiration on how this can be done.

Reply:
@blakewilson thoughts on using `wp_generate_uuid4()` for this?

Reply:
@mindctrl I think that's perfect!